kuma/payments/jinja2/payments/index.html

```diff
@@ -1,9 +1,14 @@
 {% extends "react_base.html" %}
 
-{% block title %}{{ _('Become a monthly supporter') }}{% endblock %}
+{% block title %}{{ page_title(_('Become a monthly supporter')) }}{% endblock %}
 
 {% block document_head %}
-    {{ render_react("SPA", request.LANGUAGE_CODE, request.get_full_path(), None)|safe }}
+    <!--
```

Comment:
This needs to be `{#` and `#}` otherwise it becomes a HTML comment.

Reply:
Fixed. Thank you!